spec/support/helpers/api_helper.rb

```diff
@@ -6,6 +6,7 @@ module ApiHelper
   T3_SEARCH = "#{Rails.configuration.prisoner_search_host}/prisoner-search"
   KEYWORKER_API_HOST = ENV.fetch('KEYWORKER_API_HOST')
   COMMUNITY_HOST = "#{Rails.configuration.community_api_host}/secure"
+  ELITE2LATESTMOVE = "#{T3}/movements/offenders?latestOnly=true&movementTypes=TAP"
```

Comment:
It would be nice remove the 'Elite2' reference from this variable name.

I thought we'd done a pretty good job of replacing all Elite2 references with Prison API – to avoid confusion to future devs who don't know that API's history. Looks like we missed a this in the API helper.